src/api/app/views/webui/kiwi/images/_profiles.html.haml

```diff
@@ -0,0 +1,12 @@
+.nested-fields
+  %h2
+    Profiles
```

Comment:
How about adding a little text what this does? Like: `Select one or more profiles to build. Selecting no profile means...`

Reply:
I added some text, let me know what you think.